src/main/resources/com/google/api/codegen/java/message.snip

```diff
@@ -10,12 +10,7 @@
 @end
 
 @private schemaClassDef(schema)
-  public final class {@schema.innerTypeName} implements Serializable
-      @if schema.isRequestMessage()
-        , ApiMessage {
-      @else
-        {
-      @end
+  public final class {@schema.innerTypeName} implements Serializable, ApiMessage {
```

Comment:
I think we can simplify this by assuming that `ApiMessage` implements `Serializable`.